specification/peering/resource-manager/readme.python.md

```diff
@@ -12,16 +12,16 @@ python:
   payload-flattening-threshold: 2
   namespace: azure.mgmt.peering
   package-name: azure-mgmt-peering
-  package-version: 2.0.0
+  package-version: 0.1.0
```

Comment:
why downgrade package version? has 2.0.0 released? version downgrading is confusing

Reply:
We haven't released any Python packages previously. This is the first version: https://pypi.org/project/azure-mgmt-peering/0.1.0rc